dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -333,6 +359,15 @@ private String createExtendedQueryFromAccountUsage(ConnectorArguments arguments)
                 + "WHERE end_time >= to_timestamp_ltz('%s')\n"
                 + "AND end_time <= to_timestamp_ltz('%s')\n"
                 + "AND is_client_generated_statement = FALSE\n");
+    
+    // Add database filtering if limit-to-databases is provided
```

Comment:
you can extract this block to a method - you will remove the repetition and you can use meaningful name instead of a comment